cosmic-applet-time/src/window.rs

```diff
@@ -238,19 +300,57 @@ impl cosmic::Application for Window {
             self.core.applet.anchor,
             PanelAnchor::Top | PanelAnchor::Bottom
         );
+
         let button = cosmic::widget::button(if horizontal {
-            let format = match (
-                self.config.military_time,
-                self.config.show_date_in_top_panel,
-            ) {
-                (true, true) => "%b %-d %H:%M",
-                (true, false) => "%H:%M",
-                (false, true) => "%b %-d %-I:%M %p",
-                (false, false) => "%-I:%M %p",
+            let mut elements = Vec::new();
+
+            if self.config.show_date_in_top_panel {
+                let mut date_bag = Bag::empty();
+
+                if self.config.show_day_name {
+                    date_bag.weekday = Some(components::Text::Short);
+                }
+
+                date_bag.day = Some(components::Day::NumericDayOfMonth);
+                date_bag.month = Some(components::Month::Long);
+
+                elements.push(
+                    self.core
+                        .applet
+                        .text(self.format(date_bag, &self.now))
+                        .into(),
+                );
+
+                elements.push(horizontal_space(Length::Fixed(14.0)).into())
```

Comment:
I don't believe there is meant to be a 14 pixel horizontal space. It should just be a regular space, unless I missed some discussion about this.

Reply:
Yeah, 14 was random. I don't know what is "regular space"

Reply:
Just the whitespace character.

Reply:
Ahh